internal/engine/wazevo/frontend/lower.go

```diff
@@ -3378,7 +3378,6 @@ func (c *Compiler) lowerCurrentOpcode() {
 			state.push(res)
 		case wasm.OpcodeAtomicFence:
 			order := c.readByte()
-			c.readI32u()
```

Comment:
Sorry this was just complete typo combined with not understanding the SSA test output well enough ><